R/pkg/R/SQLContext.R

```diff
@@ -744,6 +747,9 @@ read.df.default <- function(path = NULL, source = NULL, schema = NULL, ...) {
   if (is.null(source)) {
     source <- getDefaultSqlSource()
   }
+  if (source == "csv" && is.null(options[["nullValue"]])) {
```

Comment:
I think `na.strings` works for `read.table` and not just for `read.csv` in R ? Is the concern that `NA` is not a good default for other formats like JSON etc. ?